packages/fast-components-react-msft/src/select/select.schema.ts

```diff
@@ -3,8 +3,8 @@ export default {
     title: "Select",
     description: "A select component's schema definition.",
     type: "object",
-    id: "@microsoft/fast-components-react-msft/select",
-    formPluginId: "@microsoft/fast-components-react-msft/select",
+    id: "@microsoft/fast-components-react-base/select",
```

Comment:
Making sure this is correct. I'm not quite sure how these work.

Reply:
we are in the @microsoft/fast-components-react-msft package so we shouldn't be changing these to base.